### PR TITLE
Add update notification for v1.39

### PR DESCRIPTION
**Pull request by WorldLanguages**
image

### CODE REVIEW

**Comment by WorldLanguages** [comment]
v1.39.0 changelog:

```
- New addon: asset conflict dialog
- New addon: delete other costumes or sounds
- New addon: customizable code editor grid
- New addon: remaining reply count
- "Customizable block colors" addon: new insertion marker customization settings
- "Project video recorder" addon: reorganized modal and now remembers settings from last recording
- "ocular integration" addon: use newer purple accent color instead of old blue
- "Gamepad support" addon: added support for dark mode in the gamepad settings panel
- "Compact editor" addon: costume numbers in the costumes tab are now easier to read
- "Scratch Messaging" addon: show comment previews even if they aren't found
- "Scratch Messaging" addon: in the "studio activity" category, show the number of messages instead of the studio amount
- "Editor find bar" addon: fixed Ctrl+Z undoing what was typed in the find bar (Chrome/Edge only)
- Bug fix: "auto-hiding block palette" closing the palette while a block dropdown is open or editing an input
- Bug fix: "HD image uploads" not discarding EXIF metadata before uploading files
- Bug fix: sprite pane not using all available space if “compact editor” enabled and “collapsing sprite properties” uncollapsed
- Bug fix: "rearrangeable custom block inputs" not working with empty labels
- Bug fix: "costume editor snapping" not enforcing minimum and maximum values for "snapping distance" setting
- Bug fix: "cat blocks" setting "watch mouse cursor" not working for newly loaded extensions (e.g. Makey Makey)
- Bug fix: "customizable block colors" not reacting to any setting changes after creating a custom block
- Bug fix: "editor find bar" doing case-sensitive broadcast search
- Bug fix: "editor find bar" preventing Ctrl+F from opening browser find outside of the project editor
- Bug fix: "two-column category menu" scrolls up instead of down when clicking in the space below the scrollbar
- Bug fix: "data category tweaks" causing "Save now" button to always show even if no changes were made
- Bug fix: "save blocks as image" being slow due to requesting the same image multiple times from Scratch servers
- Bug fix: incompatibilities between “project progress bar” and “faster project loading”
- Bug fix: the color scheme of stage monitors (e.g. variable sliders) were controlled by website dark mode instead of editor's
- Bug fix: number pad and other editor interface elements showing above modals and popups
- Removed addon "profile statistics"
- Removed addon "local timezone on forums"
- Removed addon "forums image uploader"
```

**Comment by WorldLanguages** [comment]
I'm going to merge this tomorrow

File: _locales/en/messages.json [hunk 1]
```diff
@@ -245,18 +245,18 @@
...
   "scratchAddonsSettings": {
     "message": "Scratch Addons settings"
   },
   "sendFeedbackNotification": {
     "message": "send feedback"
   },
-  "extensionUpdateInfo2_v1_38": {
-    "message": "This update doesn't introduce any new features, but stay tuned for upcoming updates."
+  "extensionUpdateInfo2_v1_39": {
+    "message": "Other new addons added this update are \"asset conflict dialog\", \"customizable code editor grid\", and  \"remaining reply counter\"."
```
Comment (WorldLanguages) [inline _locales/en/messages.json:259]: If PR gets merged:

```suggestion
    "message": "Other new addons added this update are \"asset conflict dialog\", \"customizable code editor grid\", \"customizable stage monitors\", and  \"remaining reply counter\"."
```

Comment (WorldLanguages) [reply _locales/en/messages.json:259]: Self-reminder: also add it to changelog above.

**Review by Samq64 — APPROVED** [review]

**Comment by Samq64** [comment]
Why are the update notifications and version bump done separately?

**Comment by mxmou** [comment]
> Why are the update notifications and version bump done separately?

The notification is added earlier so that there's enough time to translate it before release.

**Pull request merged**